ci/setup-gdk.ps1

```diff
@@ -12,8 +12,8 @@ Push-Location $gdk_path
     }
     $env:MSBUILD_EXE = "`"$msbuild_path`""
     if($includeTraceLibs) {
-        cmd /c SetupIncTraceLibs.bat
+        cmd /c SetupIncTraceLibs.bat --mobile
     } else {
-        cmd /c Setup.bat
+        cmd /c Setup.bat --mobile
```

Comment:
Does this influence non mobile target?

Reply:
It will pull mobile worker_sdk libraries, but this script is specifically for CI, and our CI integration will make Android a regular part of builds, e.g. premerge, example project, and unreal engine builds will always build Android.

Reply:
So `--mobile` mean will download all libraries, not just download libraries for mobile target.

Reply:
yes, mobile means all libraries + mobile libraries